localgov_microsites_base.info.yml

```diff
@@ -11,6 +11,7 @@ libraries:
   - localgov_microsites_base/fonts
   - localgov_microsites_base/variables
   - localgov_microsites_base/component-overrides
+  - localgov_microsites_base/preview
```

Comment:
This will load the preview library on every page of the website. I thing it would be better to attach this to the page--preview template or add it via preprocess.